packages/trader/src/Modules/SmartChartBeta/Components/LaunchModal/launch-modal.tsx

```diff
@@ -0,0 +1,74 @@
+import { Button, DesktopWrapper, MobileWrapper, Modal, PageOverlay, UILoader } from '@deriv/components';
+import { LocalStore, getUrlBase } from '@deriv/shared';
+
+import { Localize } from '@deriv/translations';
+import React, { useState } from 'react';
+import './launch-modal.scss';
+import { observer } from 'mobx-react';
+import { useStore } from '@deriv/stores';
+
+const ContinueButton: React.FC<{ handleOpen: () => void }> = ({ handleOpen }) => (
```

Comment:
@ahmadtaimoor-deriv there were a lot of disputes about React.FC, i wouldn't use it unless really necessary. Here it would be enough to add type for props. And if @balakrishna-deriv is ok with React.FC here, it will be ok for me too.